src/applications/bmqtool/README.md

```diff
@@ -117,6 +117,7 @@ opened regularly (without specifying a mode).  To run the `bmqtool`, you invoke
 | `post`    | `uri=string payload=string [, ...] [async=true]` | Post a message (the `payload`) to the queue at the given `uri`. |
 | `list`    | N/A            | List the messages that have yet to be ACKed by the tool.     |
 | `confirm` | `guid=string`  | Confirm (ACK) the message matching the given GUID.           |
+| `batch-post` | `uri=string payload=string [, ...] (eventSize=N) (eventsCount=M) (postInterval=P) (postRate=R)` | Post M messages to the queue at the given `uri`. |  
```

Comment:
Also might be good to explain optional parameters.
What are default values if these params are not specified?
What are the special values for these params? What if we set `postInterval=0` or negative?
Is post interval in milliseconds?
What is the `postRate`?